src/draw/projection.typ

```diff
@@ -18,23 +14,72 @@
   matrix.transform-rotate-z(z),
 )
 
-// Pushes a view- and projection-matrix to transform all `body` elements. The current context transform is not modified.
+#let ortho-projection-matrix = (
+  (1, 0, 0, 0),
+  (0, 1, 0, 0),
+  (0, 0, 0, 0),
+  (0, 0, 0, 1),
+)
+
+// Sort drawables by median or max segment z-value
```

Comment:
It doesn't look like it sorts by median?

Reply:
Oops, forgot to remove that commen.